lib/gris/setup.rb

```diff
@@ -19,5 +19,28 @@ def env=(environment)
     def db_connection_details
       YAML.load(ERB.new(File.read('./config/database.yml')).result)[Gris.env]
     end
+
+    def cache
+      @_cache ||= ActiveSupport::Cache.lookup_store(:memory_store)
+    end
+
+    def cache=(store_option)
+      @_cache = ActiveSupport::Cache.lookup_store(store_option)
```

Comment:
That pattern was copied from https://github.com/bellycard/napa and prevents the need for something like:

`config.cache_store =`